apps/admin/frontend/src/__snapshots__/app.test.tsx.snap

```diff
@@ -215,7 +215,7 @@ exports[`L&A (logic and accuracy) flow 1`] = `
         <span
           class="c3"
         >
-          This report was created on Tuesday, November 3, 2020 at 10:22:00 PM UTC.
+          This report was created on Tuesday, November 3, 2020 at 10:22:01 PM UTC.
```

Comment:
Previously `mockdate` and `jest` where operating out of sync, so the time _should_ have been the new time (based on initial time and the `sleep` times, but it wasn't). Now they operate together.